### PR TITLE
Rlquest tf2docker

### PR DESCRIPTION
**Pull request by danideter**
Complete updating RL quest to TF2

### CODE REVIEW

File: quests/rl/hyperparam.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 trainingInput:
   scaleTier: BASIC_GPU
   hyperparameters:
-    maxTrials: 50
+    maxTrials: 30
     maxParallelTrials: 30
```
Comment (PraneetDutta) [inline quests/rl/hyperparam.yaml]: This does not take advatange of Bayesian Optimization if we set MaxParallelTrials = maxTrials. Perhaps reduce it to a lower number(with the tradeoff of completing faster)

File: quests/rl/trainer/model.py [hunk 2]
```diff
@@ -14,74 +14,36 @@
...
+    predicted Q-values and Target Q-values.
+
+    Args:
+        space_shapes ():
+        learning_rate (float): the nueral network's learning rate.
+        hidden_neurons (int): the number of neurons to use per hidden
+            layer.
+    """
+    model = tf.keras.Sequential([
+        tf.keras.layers.Dense(
```
Comment (PraneetDutta) [inline quests/rl/trainer/model.py]: nit(Optional): Perhaps number of layers can also be a parameter.

File: quests/rl/hyperparam.yaml (hunk 1 above)
Comment (danideter) [reply quests/rl/hyperparam.yaml]: Yeah, I was really heartbroken about that too. Did a lot of experimentation, and this was statistically the best number to set it at to get the agent to reach a +100 score within 20 minutes.

File: quests/rl/trainer/model.py (hunk 2 above)
Comment (danideter) [reply quests/rl/trainer/model.py]: I'll add it as an exercise for the reader :)

**Comment by danideter** [comment]
Thanks as always, Praneet!

File: quests/rl/hyperparam.yaml (hunk 1 above)
Comment (PraneetDutta) [reply quests/rl/hyperparam.yaml]: Ok, makes sense, maybe in the write up can mention this as a footnote.

**Pull request merged**